cloudigrade/account/util.py

```diff
@@ -45,22 +45,55 @@ def create_initial_aws_instance_events(account, instances_data):
     saved_instances = collections.defaultdict(list)
     for region, instances in instances_data.items():
         for instance_data in instances:
-            instance, __ = AwsInstance.objects.get_or_create(
-                account=account,
-                ec2_instance_id=instance_data['InstanceId'],
-                region=region,
-            )
+            saved_instances[region].append(
+                save_instance_events(account, instance_data, region))
+    return dict(saved_instances)
+
+
+def save_instance_events(account, instance_data, region, events=None):
```

Comment:
Should we put AWS in the function name? Just looking forward to Azure. This method clearly expects AWS.  Or we punt this to another issue to make sure we handle things more generally (when we do Azure work).  Not a blocker.

Reply:
I had the same thought, but I figure we punt this until we are actually implementing multi-cloud as it'll require a lot of other changes at the same time. A lot of these might get refactored to `util.aws` at that time.